src/components/search/results/next-queries-group.vue

```diff
@@ -1,13 +1,20 @@
 <template>
-  <div class="x-list x-padding--06 x-background--neutral-95 x-list--gap-06">
+  <div
+    class="x-list x-background--neutral-95 x-list--gap-05"
+    :class="
+      $x.device === 'mobile'
+        ? 'x-padding--05 x-padding--top-06 x-padding--bottom-06 x-margin--top-05' +
+          ' x-margin--bottom-05'
+        : 'x-padding--06 x-padding--bottom-07'
+    "
+  >
     <div class="x-list x-list--gap-03">
-      <h1 class="x-title3 x-text--bold">{{ $t('nextQueriesGroup.title') }}</h1>
       <p v-html="$t('nextQueriesGroup.message', { query: $x.query.search })" class="x-text" />
     </div>
     <NextQueries
       #suggestion="{ suggestion }"
       :suggestions="nextQueries"
-      class="x-list--gap-06"
+      class="x-list--horizontal x-list--wrap x-list--gap-03"
       :max-items-to-render="4"
```

Comment:
Did we decide on something regarding items to render on mobile and in desktop?

Reply:
we decide to wrap them like in the design for the mobile version.